aten/src/ATen/core/ATenDispatch.h

```diff
@@ -15,20 +18,34 @@
 
 namespace at {
 
+template <typename R, typename... Args>
+struct lazy;
+template <typename R, typename... Args>
+typename lazy<R, Args...>::FuncType make_lazy(
+    R (*fn)(Args...),
+    const char* schema);
```

Comment:
Question for the design of boxed dispatch key fallback: in your prototype, you save `const char* schema`. However, tracing the usages of it, it seems that what you really care about in the end is the *symbol* corresponding to the function, so you can make a JIT op for it. (1) Is this true? (2) Are there other uses of, e.g., `FunctionSchema` or similar that you anticipate needing?